src/monitors/index.js

```diff
@@ -1,4 +1,5 @@
 module.exports = [
 	require('./waitingpods'),
 	require('./longnotready'),
+  require('./newdeployment'),
```

Comment:
We're using one tab indentation. Please reindent this line and the monitor file.